requirements_all.txt

```diff
@@ -39,7 +39,7 @@ HAP-python==2.4.2
 Mastodon.py==1.3.1
 
 # homeassistant.components.isy994
-PyISY==1.1.0
+PyISY==1.1.1
```

Comment:
There should be no changes to the requirements files except for the lg_netcast module.

Reply:
I think it's ok now.
There's no change since the requirement is already here because of media_player.lg_netcast